services/wallet/activity/TODO.md

```diff
@@ -0,0 +1,157 @@
+# DB extension and refactoring for activity
```

Comment:
This is part of the bigger work package. I will remove it after finishing the filtering work.